### PR TITLE
Fix double escape in battle and browser tab

### PR DESCRIPTION
**Pull request by DaWoblefet**
Hopefully this is the correct place to remove the escape (sample replay: https://replay.pokemonshowdown.com/gen81v1-1148003675)

Before:
image

After:
image

### CODE REVIEW

**Comment by Zarel** [comment]
No, that is unfortunately a dangerous place to remove an escape. The correct place is elsewhere.

File: js/client-topbar.js [hunk 1]
```diff
@@ -91,7 +91,7 @@
 			case 'rooms':
 				return buf + ' aria-label="Join chatroom"><i class="fa fa-plus" style="margin:7px auto -6px auto"></i> <span>&nbsp;</span></a></li>';
 			case 'battle':
-				var name = BattleLog.escapeHTML(room.title);
+				var name = room.title;
```
Comment (Zarel) [inline js/client-topbar.js:94]: Specifically, this is bad and will need to be reverted.

File: js/client.js [hunk 2]
```diff
@@ -937,7 +937,7 @@ function toId() {
 					var replayLink = 'https://' + Config.routes.replays + '/' + replayid;
 					$.ajax(replayLink + '.json', {dataType: 'json'}).done(function (replay) {
 						if (replay) {
-							var title = BattleLog.escapeHTML(replay.p1) + ' vs. ' + BattleLog.escapeHTML(replay.p2);
+							var title = replay.p1 + ' vs. ' + replay.p2;
```
Comment (Zarel) [inline js/client.js:940]: This one is OK.

File: js/client-topbar.js (hunk 1 above)
Comment (Zarel) [reply js/client-topbar.js:94]:
```suggestion
				var name = BattleLog.escapeHTML(room.title);
```

Comment (DaWoblefet) [reply js/client-topbar.js]: Turns out it was also unnecessary! The other was all that was needed to get the correct results.

**Pull request merged**